### ISSUE TITLE
Question 69 - conflastion of terms

### ISSUE THREAD

**Issue by LukeStonehm** [8values/8values.github.io#37]
The question is:

> All people - regardless of factors like culture or sexuality - should be treated equally.

Culture, unlike sexuality or race, is mutable. Was this question deliberately worded to conflate a mutable attribute and an immutable attribute?

**Comment by ttaylordev** [comment]
^The immutability of sexuality is debatable. One could classify it as a part of that persons culture. I agree that there are issues with this question as well as many others like this next one: "Do you prefer a strong leader"... as opposed to a weak one? Supporting a strong leader does not make a person pro-authority at the expense of liberty. The overall aggregation methodology of users' answers is overreaching and generous in scope when combined with vague or poorly worded questions.

**Comment by hsoj95** [comment]
I agree as well to an extent. I feel this question is quite ambiguous for a number of reasons. There are potentially many possible factors and categories that this question could include. Just using culture and sexuality as definers I'm afraid serves to only muddy what the true question might be. Had the question just been about culture, this wouldn't be an issue. The issue is introducing a second factor like sexuality, along with the general wording of the question. The issue is that by introducing a second factor, the question suddenly becomes very vague as to what it's extent it. I'll give an example, under the general basis for this question, it's asking to what extent people should be treated in an egalitarian way. Here's the issue with that, where does the question end? Should we assume that wealth and/or class are included in the general basic idea of the question, suddenly we have an economic issue based question, but what about those that want to treat other cultures or sexualities in a different, non-equal,  way, but still want a "fair", egalitarian economic system? When I first read this question, I had trouble not lumping in things like wealth or class into what this question might include, prompting me to choose an answer based on that more so than things like culture or sexuality.

My suggestion to make this question clearer is to simply make the question about one specific thing, like culture, instead of including multiple different categories under it. This should help to clear up some confusion with the question
